assets/bower_components/three-js/three-js.html

```diff
@@ -54,7 +46,8 @@
           });
           Polymer.dom(this.root).appendChild(renderer.domElement);
           this.renderer = renderer;
-          renderer.setSize(this.w, this.h);
+          var parent = Polymer.dom(this).parentNode;
```

Comment:
@rohitvarkey I just added a check for this, but what do you think about setting things up so that there is *always* a div? Can you think of an example where one doesn't want a div?